### ISSUE TITLE
Xcode 9 AWSCore error when compiling

### ISSUE THREAD

**Issue by alexvazquez** [aws/aws-sdk-ios#741]
I have a Swift 3.2 code and today i download Xcode 9 and tried to compile but Im getting error in AWSCore project:

![image](https://user-images.githubusercontent.com/347747/30893471-f544e6e0-a303-11e7-9ee2-340093a56fe5.png)

Any clue?

**Comment by valerianb** [comment]
Also affected by this issue. It's preventing us from migrating to Xcode 9

**Comment by kredden** [comment]
It looks like it might be an issue with Xcode 9.  This was reported on another issue and fixed by deleting the pods and doing a "pod install".

https://github.com/awslabs/aws-sdk-ios-samples/issues/215

Regards,
Kevin

**Comment by valerianb** [comment]
Indeed, works fine with a clean install of SDK 2.6.2. Thank you Kevin and Rohan.
